pkg/postgres/walker/schema.go

```diff
@@ -97,6 +98,12 @@ func (s *SchemaRelationship) OtherSchemaColumnNames() []string {
 	return seq
 }
 
+// PermissionChecker is a permission checker that could be used by GenericStore
```

Comment:
With this we can remove https://github.com/stackrox/stackrox/blob/3a29729054029bcdfd57de3b4dfb6ad0abdebda8/pkg/search/postgres/store.go#L38C1-L42